mempool/mempool.go

```diff
@@ -185,6 +185,8 @@ func (am *AuctionMempool) RemoveWithoutRefTx(tx sdk.Tx) error {
 
 	if isAuctionTx {
 		am.removeTx(am.auctionIndex, tx)
+	} else {
```

Comment:
Im down to do that as long as we don't remove the nested bundle txs from auction txs in `RemoveTx`. We did this so that users cant get grieved by bad bids.

Reply:
Deprecated the other remove.